content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "深挖 Ambient 与 Sidecar 并存的网络流量路径"
+description: "深挖 Ambient 与 Sidecar 并存的流量路径。"
+publishdate: 2023-09-18
+attribution: "Steve Zhang (Intel), John Howard (Google), Yuxing Zeng(Alibaba), Peter Jausovec(Solo.io); Translated by Wilson Wu (DaoCloud)"
+keywords: [traffic,ambient,sidecar,coexistence]
+---
+
+Istio 有两种部署模式：Ambient 模式和 Sidecar 模式。
+前者还在持续开发中，后者是经典的网格方案。因此，Ambient 模式和 Sidecar
+模式并存应该是一种正常的部署形式，这也是这篇博客可能对 Istio 用户有所帮助的原因。
+
+## 背景 {#background}
+
+在现代微服务架构中，服务之间的通信和管理至关重要。为了应对这一挑战，
+Istio 作为一种服务网格技术应运而生。它利用 Sidecar 提供流量控制、安全性和卓越的可观察功能。
+为了进一步提高 Istio 的适应性和灵活性，Istio 社区开始探索一种新的模式 —— Ambient 模式。
+在这种模式下，Istio 不再依赖显式的 Sidecar 注入，而是通过 ztunnel 和 waypoint
+代理实现服务之间的通信和网格管理。Ambient 还带来了一系列改进，例如更低的资源消耗、更简单的部署以及更灵活的配置选项。
+启用 Ambient 模式后，我们不再需要重新启动 Pod，这使得 Istio 在各种场景中能够更好地发挥作用。
+
+有很多介绍和分析 Ambient 的博客，
+这些内容可以在本博客的[参考资源](#reference-resources)部分找到，
+本博客将分析 Istio Ambient 和 Sidecar 模式下的网络流量路径。
+
+为了阐明网络流量路径并使其更容易理解，本博文探讨了以下两种场景并配有相应的图表：
+
+- **Ambient 模式下的服务到 Sidecar 模式下服务的网络路径**
+- **Sidecar 模式下的服务到 Ambient 模式下服务的网络路径**
+
+## 关于分析的信息 {#information-about-the-analysis}
+
+分析基于 Istio 1.18.2，其中 Ambient 模式使用 iptables 进行重定向。
```

Comment:
line 31-33 has different content with en now:

```
_Note 1: The following analysis is based on Istio 1.18.2, where ambient mode uses iptables for redirection._

_Note 2: The communications between sidecar and ztunnel/waypoint proxy uses `[HTTP Based Overlay Network (HBONE)](https://docs.google.com/document/d/1Ofqtxqzk-c_wn0EgAXjaJXDHB9KhDuLe-W3YGG67Y8g/edit)`._
```

Reply:
Yes, let's review after the fixes PR merged.

Reply:
All synced.